solr/core/src/test/org/apache/solr/security/BasicAuthOnSingleNodeTest.java

```diff
@@ -45,13 +45,13 @@ public void setupCluster() throws Exception {
         .setBasicAuthCredentials("solr", "solr")
         .process(cluster.getSolrClient());
     cluster.waitForActiveCollection(COLLECTION, 4, 4);
+  }
 
-    JettySolrRunner jetty = cluster.getJettySolrRunner(0);
-    jetty.stop();
-    cluster.waitForJettyToStop(jetty);
-    jetty.start();
-    cluster.waitForAllNodes(30);
-    cluster.waitForActiveCollection(COLLECTION, 4, 4);
+  @Override
+  @After
+  public void tearDown() throws Exception {
+    cluster.shutdown();
```

Comment:
With just one test in this class, resource cleanup happened automatically at the end I guess, but with two methods, we need to shutdown cluster in `tearDown`...